src/nu/xom/tests/BaseURITest.java

```diff
@@ -658,14 +658,26 @@ public void testRelativeBaseURIsNotAllowed() {
     
     public void testRelativeURIResolutionAgainstARedirectedBase()
       throws IOException, ParsingException {
-        
+        if (CITestUtil.isRunningInCI()) {
+            // Skip network tests in CI where connectivity may be unreliable
+            return;
+        }
         Builder builder = new Builder();
-        Document doc = builder.build(
-          "http://www.ibiblio.org/xml/redirecttest.xml");
-        assertEquals(
-          "http://www.cafeconleche.org/redirecttest.xml", 
-          doc.getBaseURI()
-        );
+        try {
+            Document doc = builder.build(
+              "http://www.ibiblio.org/xml/redirecttest.xml");
+            assertEquals(
+              "http://www.cafeconleche.org/redirecttest.xml", 
+              doc.getBaseURI()
+            );
+        } catch (IOException e) {
+            if (CITestUtil.isNetworkException(e)) {
```

Comment:
we don't need this and the if (CITestUtil.isRunningInCI())  check above. I thijk keep this one and remove that one.